doc/api/permissions.md

```diff
@@ -1,4 +1,28 @@
-# Policies
+# Permissions
+
+This section exposes security features available to be adopted in a
+Node.js application. The available scopes are:
+
+* [Resource-based permissions](#resource-based-permissions)
```

Comment:
I'm not quite sure if `Resource-based permission` would improve the UX. Even accepting `resources` such as `data:`, I see `Module-based permissions` better even being not 100% accurate. Happy to see your thoughts.

Reply:
Sure, “Module-based permissions” sounds fine to me. Maybe ask at the next Security WG meeting which sounds better to most people and go with that.